### PR TITLE
chore: bump deps

### PR DESCRIPTION
**Pull request by Unix4ever**
Rekres.

### CODE REVIEW

File: upstream/upstream.go [hunk 1]
```diff
@@ -189,7 +189,7 @@ func NewListWithCmp[T Backend](upstreams []T, cmp func(T, T) bool, options ...Li
 			initialScore:      1.0,
 
 			healthcheckInterval: 1 * time.Second,
-			healthcheckTimeout:  100 * time.Millisecond,
```
Comment (utkuozdemir) [inline upstream/upstream.go]: Should we make this configurable? So other places using this wouldn't be changed.

Comment (Unix4ever) [reply upstream/upstream.go]: yep, sure

Comment (Unix4ever) [reply upstream/upstream.go]: ah wait, it's already customizable 🙃

Comment (utkuozdemir) [reply upstream/upstream.go]: oopsie :D

**Comment by Unix4ever** [comment]
Anyways this PR is good, it will only update deps and kres.

**Review by utkuozdemir — APPROVED** [review]

**Pull request merged**